policies/trademark-policy.md

```diff
@@ -1,172 +1,325 @@
+Here is what you should consider when deciding whether your use of a trademark is a nominative fair use:
+
+- Can you identify the product or service in question without using the trademark;
+- Are you avoiding a likelihood of confusion in the way that you have used the trademark; and
+- Have you used only as much as is necessary to identify the product or service.
+
+With our "Foobar Certification" example above, the person offering the certification would be allowed to say, under the nominative fair use doctrine, that she is offering "Maude's Certification for Foobar software."
+
+It is almost never the case that using a Logo will be a nominative fair use since it will be a rare case where the logo is needed for strictly informational purposes.
+
+## Proper trademark use
+
+These rules hold true for all trademarks, not just ours, so you should follow them for our Marks as well as anyone else's.
+
+### Use of trademarks in text
+
+- Always distinguish trademarks from surrounding text with at least initial capital letters or in all capital letters.
+    - Unacceptable: hibernate
+    - Acceptable: Hibernate, HIBERNATE
+
+- Always use trademarks in their exact form with the correct spelling, neither abbreviated, hyphenated, or combined with any other word or words.
+    - Unacceptable: Open Rewrite, open rewrite, open-rewrite, openrewrite
+    - Acceptable: OpenRewrite
+    - See our [Trademark List][] to verify exactly how the marks should appear.
+
+- Don't pluralize a trademark.
+    - Unacceptable: I have seventeen JBangs running in my lab.
+    - Acceptable: I have seventeen JBang catalogs running in my lab.
+
+- Don't use "a" or "the" to refer to an instance of the trademark.
+    - Unacceptable: I put a JBang on my mom's computer.
+    - Acceptable: I put a JBang catalog on my mom's computer.
+
+- Don't use a trademark as a possessive.
+   The following noun should be used in possessive form or the sentence reworded so there is no possessive.
+   An exception to this can be when CF is used to refer to the Foundation‘s name.
+
+    - Unacceptable: OpenRewrite's unique properties ...
+    - Acceptable: Unique capability of OpenRewrite™ ...
+    - Acceptable: Commonhaus Foundations‘s IP Policy ...
+
+- Don't use a trademark as a verb. Trademarks are products or services, never actions.
+    - Unacceptable: I JReleasered my computer today!
+    - Acceptable: I installed JReleaser on my computer today!
+
+- Always use a trademark as an adjective modifying a noun. You can see the nouns we prefer on our [Trademark List][]
+    - Unacceptable: This is an OpenRewrite. Anyone can use it.
+    - Acceptable: This is a OpenRewrite recipe. Anyone can use it.
+
+- Don't translate a trademark into another language.
+    - Acceptable: Quiero instalar Hibernate en mi proyecto.
+    - Unacceptable: Quiero instalar hibernar en mi proyecto.
+
+### Use of Logos
+
+Use only official versions of the Logo obtained from the Commonhaus Foundation or its projects.
```

Comment:
Unnecessarily prescriptive. For example, there is a 3d form of the OpenRewrite logo we use (and paid significantly) for that would now run afoul of this.

<img width="1014" alt="image" src="https://github.com/user-attachments/assets/f86580e0-a688-4c74-ba75-43f02e3c997f">

Reply:
In the general case, yes, that is exactly what we want to say is not permitted. 

What we should be doing in the specific case of Commonhaus, OpenRewrite, and Moderne is something else (per your point below).

Reply:
Again, I think this falls in the "specific written agreement between Commonhaus and Moderne" category.

Reply:
Capitalize Projects to recognize those as sub-entities.

```suggestion
Use only official versions of the Logo obtained from the Commonhaus Foundation or its Projects.
```

All Projects should ideally have an artwork listing somewhere (official Logos including light mode, dark mode, and monotone). 
This clause states folks should only choose Logos from that list, and shouldn't modify them.

It does not pose any restrictions on the Project for how logos are created or added to that list. OpenRewrite (as a Commonhaus Project in a future state) is in control of where those Logos come from, which would include those designed by Moderne.